src/kafkit/ssl.py

```diff
@@ -0,0 +1,62 @@
+"""Support for connecting to brokers with SSL."""
+
+__all__ = ["create_ssl_context", "concatenate_certificates"]
+
+import ssl
+from pathlib import Path
+from typing import List
+
+
+def create_ssl_context(
+    *, cluster_ca_path: Path, client_cert_path: Path, client_key_path: Path
+) -> ssl.SSLContext:
+    """Create an SSL context for a client connecting to secured Kafka brokers.
+
+    Parameters
+    ----------
+    cluster_ca_path : `pathlib.Path`
+        Path to the cluster CA certificate.
+    client_cert_path : `pathlib.Path`
+        Path to the client certificate.
+    client_key_path : `pathlib.Path`
+        Path to the client key.
+
+    Returns
+    -------
+    ssl.SSLContext
```

Comment:
developer.lsst.io says that this will be a syntax error and you have to specify a name for the return variable along with the type. Is that documentation out of date?

Reply:
I'm experimenting with some alternative Sphinx extension set ups, so don't worry about this right now.